ClassesHomework/Animal.cs

```diff
@@ -7,8 +7,54 @@
 namespace ClassesHomework
 {
     // write your Animal class here. 
-    //class Animal
-    //{
-        
-    //}
+    public class Animal
+    {
+        public Animal()
+        {
+
+        }
+        public Animal(string breed)
+        {
+            Breed = breed;
+
+        }
+        public Animal(int age)
+        {
+            Age = age;
+
+        }
+        public Animal(string breed, int age)
+        {
+            Breed = breed;
+            Age = age;
+
+        }
+        public bool Sit()
+        {
+            IsSitting = true;
+            return IsSitting;
+        }
+        public bool Stand()
+        {
+            IsSitting = false;
+            return IsSitting;
+        }
+
+        public int CelebrateBirthday()
+        {
+            Age = ++Age;
+            return Age;
+        }
+
+        public virtual string Speak()
+        {
+            return "What am I?";
+        }
+        public virtual string Species { get; set; } = "unknown";
```

Comment:
Nothing in the tests specifically says whether Species should be publicly settable, but is this the correct access level or this property?